src/main/java/org/medicmobile/webapp/mobile/MedicAndroidJavascript.java

```diff
@@ -204,6 +206,21 @@ public void simprints_reg(int targetInputId) {
 		}
 	}
 
+	/**
+	 * @param id id associated with this message, e.g. a pouchdb docId
+	 * @param destination the recipient phone number for this message
+	 * @param content the text content of the SMS to be sent
+	 */
+	@org.xwalk.core.JavascriptInterface
+	@android.webkit.JavascriptInterface
+	public void sms_send(String id, String destination, String content) {
+		try {
+			smsSender.send(id, destination, content);
+		} catch(Exception ex) {
+			logException(ex);
```

Comment:
Good point.  We'd likely want to _also_ log it in Android.

Reply:
... although if the send then becomes async, we'll want to call smsStatusUpdate() with the exception details instead of literally throwing the exception back to javascript.